spec/lib/stupidedi/transaction_sets/005010/X212-HN277_spec.rb

```diff
@@ -0,0 +1,36 @@
+describe "Stupidedi::TransactionSets::FiftyTen::Implementations::X212::HN277" do
+  using Stupidedi::Refinements
+  include TreeMatchers
+  include NavigationMatchers
+
+  describe "use with a basic document" do
+    let(:fixdir) { "005010/X212 HR277 Heatlh Care Information Status Notification" }
+    let(:parser) { Fixtures.parse!("#{fixdir}/pass/1.era").head }
+
+    describe "parser" do
+      subject { parser }
+
+      it { is_expected.to be_deterministic }
+      it { is_expected.to be_last }
+      it do
+        is_expected.to have_separators(
+          element:    "*",
+          component:  ":",
+          repetition: "^",
+          segment:    "~"
+        )
+      end
+    end
+
+    describe "structure" do
```

Comment:
This spec isn't needed -- a mostly equivalent spec is automatically generated by virtue of having your fixture file matching the naming convention. See [this](https://github.com/irobayna/stupidedi/blob/master/spec/lib/stupidedi/transaction_sets_spec.rb#L49) for details.

However, you can still add specs here similar to [this](https://github.com/irobayna/stupidedi/blob/master/spec/lib/stupidedi/transaction_sets/005010/X221-HP835_spec.rb#L51) which ensures your grammar lets you navigate correctly. Doing so can potentially catch errors like [this](https://github.com/irobayna/stupidedi/issues/155#issuecomment-456585102) -- that comment shows a quick way to check the grammar, but it's not automated.

If you don't want to create a test that uses `have_structure`, you can delete this spec file.

Reply:
Do you mean the entire spec file is redundant, or just this particular structure testing section?

Reply:
I think both are redundant now that you mention it. Parts of these are spread over different tests elsewhere, though I might need to review the other specs to be certain. Either way, I intend for these two tests to be boilerplate that you don't have to write.